src/tennisSimulator/Menu.java

```diff
@@ -0,0 +1,72 @@
+package tennisSimulator;
+
+import java.util.Scanner;
+
+public class Menu {
+
+    private static PlayersDatabase playersDatabase = new PlayersDatabase();
+    private static Scanner scanner = new Scanner(System.in);
```

Comment:
Yes, you are right, those fields should be final, we don't expect them to change after initialization.